omc3/optics_measurements/data_models.py

```diff
@@ -63,13 +64,24 @@ def __init__(self, files_to_analyse: Sequence[str|Path|tfs.TfsDataFrame], optics
             self[plane] = dpp.append_dpp(self[plane], dpp.arrange_dpps(dpp_values))
             self[plane] = dpp.append_amp_dpp(self[plane], amp_dpp_values)
 
-    @staticmethod  # TODO later remove
+    @staticmethod
     def _repair_backwards_compatible_frame(df, plane: str):
         """
-        Multiplies unscaled amplitudes by 2 to get from complex amplitudes to the real ones.
-        This is for backwards compatibility with Drive,
-        i.e. harpy has this
+        Multiplies unscaled amplitudes by 2 for old files without the AMPLITUDE_UNIT header.
+        This is for backwards compatibility with Drive.
+
+        New harpy files (with AMPLITUDE_UNIT header set to 'm') don't need this correction
+        as they already output amplitudes in the correct unit.
         """
+        # Check if the file has the new AMPLITUDE_UNIT header
```

Comment:
It will soon not be "new". Maybe it you want to convey that, specify the `omc3` version it came with?